tasks/38_max-product/max-product.js

```diff
@@ -0,0 +1,10 @@
+function maxProduct(arr) {
+  let maxValue = Number.MIN_SAFE_INTEGER;
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] * arr[i + 1] > maxValue) {
```

Comment:
You are looping over the array but took `arr[i + 1]` in this line, which seems to be undefined and `arr[i] * arr[i + 1]` going to be `NaN`. I think you should go until `arr.length -1`

Reply:
yes, I understood you